src/cluster/src/install.rs

```diff
@@ -457,6 +485,29 @@ pub struct ClusterInstaller {
 }
 
 impl ClusterInstaller {
+    /// Creates a `ClusterInstaller` with default installation settings
+    ///
+    /// This will fail if there is trouble connecting to Kubernetes or
+    /// if the `helm` executable is not locally installed.
+    ///
+    /// To customize the installation settings, use the [`builder()`]
+    /// function to create a [`ClusterInstallerBuilder`].
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use fluvio_cluster::ClusterInstaller;
+    /// let installer = ClusterInstaller::new().unwrap();
+    /// async_std::task::block_on(installer.install_fluvio()).unwrap();
+    /// ```
+    pub fn new() -> Result<Self, ClusterError> {
+        Ok(Self {
+            config: ClusterInstallerBuilder::new(),
```

Comment:
It is weird that ClusterInstaller has dependencies to ClusterInstallBuilder.   It should be other way around?

Reply:
I think sometimes it is laid out that way, but something I like about this pattern is that the `ClusterInstallerBuilder` is pretty much nothing more than a config object that is put together with builder methods where you can do validation along the way. Also, by using the `ClusterInstaller::builder() -> ClusterInstallerBuilder` method, users don't actually need to import the `ClusterInstallerBuilder` type.